### ISSUE TITLE
Missing ray module in `run.sh --train`

### ISSUE THREAD

**Issue by laxatives** [AIcrowd/neurips2020-procgen-starter-kit#11]
```
$ ./run.sh --train
           _____                          _
     /\   |_   _|                        | |
    /  \    | |  ___ _ __ _____      ____| |
   / /\ \   | | / __| '__/ _ \ \ /\ / / _  |
  / ____ \ _| || (__| | | (_) \ V  V / (_| |
 /_/    \_\_____\___|_|  \___/ \_/\_/ \__,_|

Executing: python train.py -f experiments/impala-baseline.yaml --ray-memory 1500000000 --ray-num-cpus 2 --ray-object-store-memory 1000000000
Traceback (most recent call last):
  File "train.py", line 17, in <module>
    from callbacks import CustomCallbacks
  File "/Users/bhan/src/go/src/github.com/lyft/neurips2020-procgen-starter-kit/callbacks.py", line 9, in <module>
    from ray.rllib.agents.callbacks import DefaultCallbacks
ModuleNotFoundError: No module named 'ray.rllib.agents.callbacks'
```

`pip freeze` shows `ray==0.8.4`

Installed to a conda environment using 

```
pip install ray[rllib]==0.8.4
pip install tensorflow==2.1.0 # or tensorflow-gpu
pip install procgen==0.10.2
```

**Comment by laxatives** [comment]
Also, this framework is amazing. Thanks for setting up this competition.

**Comment by skbly7** [comment]
Please use ray version 0.8.5 and it should solve the issue, README in now fixed via #13.
In case you continue to face this issue let us know. 😄
